andela/heimdal-ah-backend
# #162171016 Refactor email verification functionality 162171016

#### What does this PR do?
- This PR slightly refactors the email verification functionality: It removes the expiry time from the email token being sent to the user. It also abstracts the email verification ```body``` and ```subject``` into a method called sendVerificationMail inside helper/mailer.js

#### Description of Task to be completed?
- Remove expiry time from email token
- Abstract the email verification body, subject and process to a method inside mailer.
- Make the user model email field a boolean type

#### How should this be manually tested?
- Clone the repository
- Start the server ```npm run start:dev```
- Signup on the app using postman from this endpoint (POST) ```localhost:4000/api/v1/auth```
- Expect a verification mail
- Click the link to verify the email you registered from your mail box

#### What are the relevant pivotal tracker stories?
#162171016
#### Any background context you want to add?
Nill
#### Screenshots
Nill
